bika/lims/subscribers/dep_cookie.py

```diff
@@ -32,21 +32,25 @@ def SetDepartmentCookies(event):
                                 getUsername=username)
             if not brain:
                 # It is possible that current user is created by Plone ZMI and
-                # it is not a LabContact. In this case we will just disable
-                # department filtering.
-                logger.warn("No lab Contact found... Log in with Plone user. "
+                # it is not a LabContact. We disable filtering by department because
+                # it can be Client Contact. Also we log it as a warning message in case
+                # it is not a Client Contact but a Zope User.
+                logger.warn("No lab Contact found... Plone user or Client Contact logged in. "
```

Comment:
Loggin in as a Contact shouldn't be a warning. First we should check which type of user has logged in.